tools/lsp/common.rs

```diff
@@ -70,8 +71,20 @@ impl ElementRcNode {
         })
     }
 
-    pub fn is_layout(&self) -> bool {
-        self.with_element_debug(|_, l| l.is_some())
+    pub fn layout_kind(&self) -> crate::preview::ui::LayoutKind {
+        use crate::preview::ui::LayoutKind;
```

Comment:
this fails to compile when the preview feature is not enabled.
In fact, i think the ElementRcNode don't belong in the common module but somewhere within the preview module.

Reply:
ok